redis-om-spring/src/main/java/com/redis/om/spring/convert/BytesToLocalDateConverter.java

```diff
@@ -7,17 +7,12 @@
 
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.data.convert.ReadingConverter;
-import org.springframework.util.ObjectUtils;
 
 @ReadingConverter
 public class BytesToLocalDateConverter implements Converter<byte[], LocalDate> {
 
   @Override
   public LocalDate convert(byte[] source) {
-    if (ObjectUtils.isEmpty(source)) {
-      return null;
-    }
-
     return LocalDate.ofInstant(Instant.ofEpochSecond(Long.parseLong(toString(source))), ZoneId.systemDefault());
```

Comment:
if null is still possible, shouldn't this return an Option<T>?

Reply:
No, after  checking the class that manipulates these (in Spring Data Redis) it only uses them with a non-null byte source